derive_client/bridge/client.py

```diff
@@ -25,6 +37,7 @@ def __init__(self, w3: Web3, account: Account, chain_id: ChainID):
         self.account = account
         self.chain_id = chain_id
         self.bridge_contract: Contract | None = None
+        self.withdraw_wrapper_contract: Contract | None = None
```

Comment:
this is not ideal, since:
- `load_bridge_contract` sets the `bridge_contract` needed for `deposit()`, 
- `load_withdraw_wrapper` sets the `withdraw_wrapper_contract` needed for `withdraw_with_wrapper()`

this is not great, but for now we can keep it and mark this `bridge` module as private (`_bridge`); users should only interface with the `BaseClient`, which where we call this internally, and with the cli.